scripts/sct_check_dependencies.py

```diff
@@ -223,20 +223,28 @@ def main():
             install_software = 1
 
     # CHECK DEPENDENT MODULES (installed by nibabel/dipy):
-    print_line('Check if numpy is installed')
+
+    # CHECK IF NUMPY IS INSTALLED
     try:
-        importlib.import_module('numpy')
+        numpy = importlib.import_module('numpy')
+        print_line('Check if numpy (' + numpy.version.version + ') is installed')
         print_ok()
     except ImportError:
+        print_line('Check if numpy is installed')
         print_fail()
         install_software = 1
-    print_line('Check if scipy is installed')
+
```

Comment:
If we're only looking for numpy's presence, then it would be more intuitive to have something like:
```
Check if numpy is installed.... OK (version)
```

Also, for the other packages, a specific version is looked for (arguably as a way to ensure reproducibility of results), maybe we'd want to be consistent here.

Reply:
I understand your point. But doing that would break the continuity of the way it is done in the previous lines from the point of view of what is shown.